docs/upgrade.html

```diff
@@ -69,12 +69,17 @@
         producer's <code>batch.size</code> configuration.</li>
     <li>GC log rotation is enabled by default, see KAFKA-3754 for details.</li>
     <li>Deprecated constructors of MetricName and Cluster classes have been removed.</li>
+    <li>A new Headers interface has been introduced to provide headers read and write access.</li>
```

Comment:
This cause confusion between user headers and system headers. Would the following be less ambiguous? 
"Added user headers support through a new Headers interface."